src/browserWindow.ts

```diff
@@ -76,14 +76,11 @@ export class BrowserWindow {
                 window.showTextDocument(document);
               },
               () => {
-                shell.default(filepath).then(
-                  () => {
-                    // do nothing
-                  },
-                  () => {
-                    window.showErrorMessage(`Could not open: ${filepath}`);
-                  }
-                );
+                if (!existsSync(filepath)) {
+                  window.showErrorMessage(`File does not exist: ${filepath}`);
+                } else {
+                  shell.default(filepath);
```

Comment:
why is this importing `*` as `shell` and `default`? why not just `import open from "open"` and `open.open(filepath)`?

Reply:
This is what I got when I imported the module you referenced.

Reply:
from https://github.com/sindresorhus/open#usage, I think you can just: 
```js
import open from 'open';

// Opens the URL in the default browser.
open(filepath);
```

Reply:
That worked.  Six ways to import, none of them feel right...